orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/WaitForManifestStableTask.java

```diff
@@ -63,19 +64,61 @@ public long getTimeout() {
   public TaskResult execute(@Nonnull Stage stage) {
     String account = getCredentials(stage);
     Map<String, List<String>> deployedManifests = manifestNamesByNamespace(stage);
-    List<String> messages = new ArrayList<>();
-    List<String> failureMessages = new ArrayList<>();
-    List<Map<String, String>> stableManifests = new ArrayList<>();
-    List<Map<String, String>> failedManifests = new ArrayList<>();
-    List warnings = new ArrayList<>();
+
+    List<String> messages =
```

Comment:
The pattern we've started using to avoid all this casting is to define a model for the stage context, then use `stage.mapTo()` at the beginning of the stage to get the model.  (A good recent example is `DeployManifestTask`.)  I think the model here should be fairly easy to write, just `messages`, `stableManifests`, `failedManifests`, `warnings`, and `exception` and then all this casting would go away.